pragma-ui/src/lib/constants.ts

```diff
@@ -0,0 +1,7 @@
+export const DEFAULT_ASSETS = [
```

Comment:
when do we use that ?

Reply:
Not yet usable, but we will use them when we return default assets.

Reply:
Now used it in getTokens